src/main/java/at/ac/tuwien/kr/alpha/solver/ThriceTruth.java

```diff
@@ -57,6 +57,13 @@ public static ThriceTruth valueOf(boolean value) {
 	}
 
 	/**
+	 * @return true if this is MBT.
+	 */
+	public boolean isMBT() {
```

Comment:
Why not just let users of this type check `x == MBT`? That's absolutely reasonable and clear. What is the reasoning behind adding `isMBT`?